### PR TITLE
SRVCOM-1296: Update serverless docs IDs

### PR DESCRIPTION
**Pull request by abrennan89**
For versions 4.6+

https://issues.redhat.com/browse/SRVCOM-1296

Direct link to doc preview: mostly N/A since this PR just updates IDs (no content changes)
- https://deploy-preview-33047--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/removing-openshift-serverless.html#serverless-deleting-crds is also now a module inc.
- https://deploy-preview-33047--osdocs.netlify.app/openshift-enterprise/latest/serverless/integrations/gpu-resources.html is now improved, content split into a module inc.

QE signed off by: @alanfx

### CODE REVIEW

**Comment by alanfx** [comment]
This PR is `qe-verified`. :+1:

File: modules/serverless-deleting-crds.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+//  * serverless/admin_guide/removing-openshift-serverless.adoc
+
+[id="serverless-deleting-crds"]
```
Comment (bergerhoffer) [inline modules/serverless-deleting-crds.adoc:5]: Needs the "_{context}" to the end of the ID

File: serverless/integrations/gpu-resources.adoc [hunk 2]
```diff
@@ -23,20 +23,19 @@ You can specify a GPU resource requirement when you create a Knative service usi
...
-
 +
 A GPU resource requirement limit of `1` means that the service has 1 GPU resource dedicated.
 Services do not share GPU resources. Any other services that require GPU resources must wait until the GPU resource is no longer in use.
 +
 A limit of 1 GPU also means that applications exceeding usage of 1 GPU resource are restricted.
 If a service requests more than 1 GPU resource, it is deployed on a node where the GPU resource requirements can be met.
 
-.Updating GPU requirements for a Knative service using `kn`
+[id="gpu-resources-updating-requirements"]
```
Comment (bergerhoffer) [inline serverless/integrations/gpu-resources.adoc:37]: This whole assembly is a bit weird. There is a procedure above it that should be in it's own module, and then it seems like this section should be its own module too (with the .Procedure heading, etc. - and the command indented properly under the step). 

A bit out of scope for what you're doing here, other than the fact that you moved some other content into its own module. But it's fine if you want to take a look at it for a future update!

**Review by bergerhoffer — COMMENTED** [review]
Just one thing and then a question!

File: serverless/integrations/gpu-resources.adoc (hunk 2 above)
Comment (abrennan89) [reply serverless/integrations/gpu-resources.adoc]: Made some modifications to improve this, lmk what you think 🙂 thanks for the suggestion!

**Comment by bergerhoffer** [comment]
LGTM!

**Comment by abrennan89** [comment]
Will do manual CPs

**Pull request merged**